data/map.txt

```diff
@@ -27844,6 +28041,22 @@ planet Lichen
 		threshold 3000
 		fleet "Large Militia" 8
 
+planet Lightstead
+	attributes station hai retirement
+	landscape land/space6
+	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. The reception area is lavish with several artworks and exotic plants. A well-groomed Hai receptionist informs you that, unless you have family at this respite facility, you will not be permitted to wander the station.`
+	description `The spaceport is small but serviceable, and has what looks like a luxuriously appointed Centipede parked in a berth, presumably for entertainment voyages. There is a small but fancy cafe, and a gift shop with things like gold watches, jewelery, wines and other pricey trinkets. This place provides aged care respite only for the very wealthy.`
```

Comment:
```suggestion
	description `The spaceport is small but serviceable, and has what looks like a luxuriously appointed Centipede parked in a berth, presumably for entertainment voyages. There is a small but fancy cafe, and a gift shop with gold watches, jewellery, wines and other pricey trinkets. This place provides aged care respite for only the very wealthy.`
```

Reply:
My notepad's US Spellchecker disagrees with you on jewellery (which is the British/Australian spelling I wrote the first time. :P)